src/sr_od/context/sr_pc_controller.py

```
@@ -34,6 +34,21 @@ def __init__(self, game_config: GameConfig,
 
+    def close_game(self):
+        """
+        关闭游戏
+        :return:
+        """
+        win = self.game_win.get_win()
+        if win is None:
+            return
+        try:
+            import subprocess
+            subprocess.run(["taskkill", "/f", "/fi", f"WINDOWTITLE eq {win.title}"], check=True, capture_output=True)
```

Ruff (0.14.10)

`subprocess` call: check for execution of untrusted input

(S603)